trento/xml/systemd-install.xml

```diff
@@ -706,7 +706,7 @@ server {
     listen 443 ssl;
     server_name trento.example.com;
 
-    ssl_certificate /etc/ssl/certs/trento.crt;
+    ssl_certificate /etc/nginx/ssl/certs;
```

Comment:
Same as in the markdown, we must be specific:
`ssl_certificate /etc/nginx/ssl/certs/trento.crt; `